far/FarEng.hlf.m4

```diff
@@ -3225,58 +3225,52 @@ string is changed to that of the found string. For example:
     └────────────────┴────────────────────┴──────────────────────┘
 
 @FindAllMenu
-$ #Редактор: меню с результатами поиска всех вхождений#
-    В этом меню доступны следующие сочетания клавиш:
+$ #Editor: All matching entries menu#
+  The following key combination are available in this menu:
 
-      #F5#                   - переключить размер меню
+    #F5#              - Toggle menu size.
 
-      #Ctrl+Up, Ctrl+Down#   - скроллировать текст редактора
+    #Ctrl+Up#         - Scroll the text in the editor.
+    #Ctrl+Down#
 
-      #Ctrl+Enter#           - перейти к позиции найденного текста
+    #Ctrl+Enter#      - Go to the position of the found text.
+    #Ctrl+Left#
+      #mouse click#
 
-      #Add#                  - добавить стековую закладку с текущей позицией
+    #Gray +#          - ^<wrap>Add stack bookmark with the current position.
```

Comment:
What are "stack bookmarks"? There seems to be full-fledged backend functionality in the editor, but I could not find, either in the code or in the help, any UI to access it except for this place in the Find-all menu.

Reply:
'stack bookmarks' is an outdated name. See [session bookmarks](https://api.farmanager.com/ru/service_functions/editorcontrol.html#ECTL_ADDSESSIONBOOKMARK).

Reply:
It's all about the API. However, there seems to be no built-in UI in FAR other than this single place in the Find-all dialog. Feels strange and asymmetrical. It seems without plugins or macros, one can set a session bookmark, but cannot use it in any way. I would humbly suggest removing this hotkey here as well.

Reply:
Yes, session bookmarks initially were intentionally introduced without any UI.
This feature was added [later](https://bugs.farmanager.com/view.php?id=2096), not sure why.

Reply:
Thank you. Strange.